addons/l10n_pe/data/account_tax_data.xml

```diff
@@ -224,6 +228,67 @@
             }),
         ]"/>
     </record>
+    <record id="sale_tax_exp" model="account.tax.template">
+        <field name="chart_template_id" ref="pe_chart_template"/>
+        <field name="name">0% EXP</field>
+        <field name="description">EXP</field>
+        <field name="l10n_pe_edi_tax_code">9995</field>
+        <field name="l10n_pe_edi_unece_category">S</field>
+        <field name="amount">0</field>
+        <field name="type_tax_use">sale</field>
+        <field name="sequence">1</field>
+        <field name="include_base_amount">1</field>
+        <field name="tax_group_id" ref="tax_group_exp"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 0,
+                'repartition_type': 'base',
+            }),
+
+            (0,0, {
+                'factor_percent': 0,
+                'repartition_type': 'tax',
+                'account_id': ref('chart40111'),
+            }),
+
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 0,
+                'repartition_type': 'base',
+            }),
+
+            (0,0, {
+                'factor_percent': 0,
+                'repartition_type': 'tax',
+                'account_id': ref('chart40111'),
+            }),
+        ]"/>
+    </record>
+    <record id="sale_tax_ivap" model="account.tax.template">
+        <field name="chart_template_id" ref="pe_chart_template"/>
+        <field name="name">4% IVAP</field>
+        <field name="description">IVAP</field>
+        <field name="l10n_pe_edi_tax_code">1016</field>
+        <field name="l10n_pe_edi_unece_category">S</field>
+        <field name="amount">4</field>
+        <field name="type_tax_use">sale</field>
+        <field name="sequence">1</field>
+        <field name="include_base_amount">1</field>
+        <field name="tax_group_id" ref="tax_group_ivap"/>
+    </record>
+    <record id="sale_tax_icbper" model="account.tax.template">
+        <field name="chart_template_id" ref="pe_chart_template"/>
+        <field name="name">0.2% ICBPER</field>
```

Comment:
@umiphos Why is this percentage and the amount is fixed?

Reply:
Another way to handle this is to put a type price included 100%, such that users just create a plastic bag product with a price of 0.2 as the product is taxed entirely.  @umiphos @nhomar

Reply:
We will do it on this way that you mentioned.

Reply:
@jco-odoo 

Sorry I think incomplete my previous answer.

The tax is over any plastic bag, then if I sell a plastic bag on 2 the price will be 2*1.18(VAT)+0.2==2.56 then create a product do not make much sense.

FYI: the change we did with your proposal I did not test it with multiple taxes per line and it is broken, I will revert it temporarily to show you on the next meeting the effect.

Reply:
> @umiphos Why is this percentage and the amount is fixed?

We will fix that it is a typo!